assets/js/components/dashboard-sharing/DashboardSharingSettingsButton.js

```diff
@@ -95,6 +101,15 @@ export default function DashboardSharingSettingsButton() {
 		}
 	}, [ dialogOpen, haveSettingsChanged, rollbackSharingSettings ] );
 
+	const triggeredTourRef = useRef();
```

Comment:
Used a ref here because we don't need a re-render when this changes, it's only to prevent triggering a second time.